client/src/components/Navbar.js

```diff
@@ -74,9 +79,11 @@ const Navbar = ()=>{
             setSocket({socket:socket});
         }
         if(newMsg){
-            setNotification({ messages:[...notification.messages,newMsg] });
+			const tmp = [...notification.messages,newMsg].slice(-5);
```

Comment:
What is this slice(-5) about?
Also for the future: in general I think it's better to avoid variable names called temp or copy. Mostly because they're not descriptive or they show that the variable is unnecessary. Even though it's longer, I think it's better to call the variable name something like messagesWithoutLastFive (ideally instead of "last five" describe what you're taking out) random person reading the code knows what's going on

Reply:
Got it!